tracer/src/Datadog.Trace/PDBs/MethodSymbolResolver.cs

```diff
@@ -178,6 +180,16 @@ public bool TryGetMethodSymbol(MethodInfo methodInfo, out MethodSymbol methodSym
                         {
                             moduleDef = mDef;
                         }
+                        else if (options.PdbFileOrData is null && !string.IsNullOrEmpty(module.Assembly.Location))
+                        {
+                            // If the PDB cannot be loaded automatically we do a final try to manually find it.
+                            var pdbFile = Path.ChangeExtension(module.Assembly.Location, "pdb");
```

Comment:
From [the docs](https://learn.microsoft.com/en-us/dotnet/api/system.reflection.assembly.location?view=net-6.0#remarks):

> .NET Framework only: If the loaded file was [shadow-copied](https://learn.microsoft.com/en-us/dotnet/framework/app-domains/shadow-copy-assemblies), the location is that of the file after being shadow-copied. To get the location before the file has been shadow-copied, use the [CodeBase](https://learn.microsoft.com/en-us/dotnet/api/system.reflection.assembly.codebase?view=net-6.0) property.

Reply:
But also from [the docs for `Assembly.CodeBase`](https://learn.microsoft.com/en-us/dotnet/api/system.reflection.assembly.codebase?view=net-6.0):

> `Assembly.CodeBase` and `Assembly.EscapedCodeBase` are only included for .NET Framework compatibility. Use `Assembly.Location`.

😅

Reply:
![image](https://user-images.githubusercontent.com/69803/195916020-243adfb3-0d4a-42a2-9da2-223afe582010.png)